autoload/neomake/makers/ft/elixir.vim

```diff
@@ -1,12 +1,34 @@
 " vim: ts=4 sw=4 et
 
+" Credo error types.
+" F -> Refactoring opportunities
+" W -> Warnings
+" C -> Convention violation
+" D -> Software design suggestions
+" R -> Readability suggestions
+" Map structure {CredoError: NeomakeType, ...}
+let s:neomake_elixir_credo_type_map = {
+    \ 'F': 'W',
+    \ 'C': 'W',
+    \ 'D': 'I',
+    \ 'R': 'I'}
+
 function! neomake#makers#ft#elixir#PostprocessEnforceMaxBufferLine(entry) abort
     let buffer_lines = str2nr(line('$'))
     if (buffer_lines < a:entry.lnum)
         let a:entry.lnum = buffer_lines
     endif
 endfunction
 
+function! neomake#makers#ft#elixir#PostprocessCredo(entry) abort
+    let type = toupper(a:entry.type)
+    let type_map = extend(s:neomake_elixir_credo_type_map,
+                \ get(g:, 'neomake_elixir_credo_type_map', {}))
```

Comment:
> Regarding the variable name, we might was to use something more namespacy, e.g. g:neomake_elixir_credo_config_typemap, i.e. _config for anything that is maker specific.

I agree.  The names should have a predictable pattern.  I actually would prefer maker configs to be dicts since that's a pretty long name, but it's a little late for that now.

> /cc @tweekmonster

Because I'm super opinionated about small details right?!  😀